src/js/tabs/motors.js

```diff
@@ -726,7 +737,56 @@ TABS.motors.initialize = function (callback) {
         // enable Status and Motor data pulling
         GUI.interval_add('motor_and_status_pull', get_status, 50, true);
 
-        GUI.content_ready(callback);
+        let zeroThrottleValue = rangeMin;
+
+        if (self.feature3DEnabled) {
+            zeroThrottleValue = neutral3d;
+        }
+
+        setup_motor_output_reordering_dialog(content_ready, zeroThrottleValue);
+
+        function content_ready() {
+            GUI.content_ready(callback);
+        }
+
+       GUI.content_ready(callback);
+    }
+
+    function setup_motor_output_reordering_dialog(callbackFunction, zeroThrottleValue)
+    {
+        const domMotorsEnableTestModeCheckbox = $('#motorsEnableTestMode');
+        const domDialogMotorOutputReorder = $('#dialogMotorOutputReorder');
+
+        const motorOutputReorderComponent = new MotorOutputReorderComponent($('#dialogMotorOutputReorderContent'),
+            callbackFunction, mixerList[FC.MIXER_CONFIG.mixer - 1].name,
+            zeroThrottleValue, zeroThrottleValue + 200);
+
+        $('#dialogMotorOutputReorder-closebtn').click(closeDialog);
+
+        function closeDialog()
+        {
+            domMotorsEnableTestModeCheckbox.prop('checked', false);
+            mspHelper.setArmingEnabled(false, false);
```

Comment:
Not sure why this is in here - I don't think this functionality should have to interact with the arming disabled status at all.

Reply:
@mikeller , `domMotorsEnableTestModeCheckbox` is a checkbox in the new dialog
However youa re right about `mspHelper.setArmingEnabled(false, false);` it should not be here. I will remove it.

Reply:
removed